src/test/resources/pages/adminActivityLogPage.html

```diff
@@ -671,11 +552,11 @@ <h4 class="text-center">
 
 
 
-    <div class="alert alert-warning" id="statusMessage">
+    <div class="alert alert-info" id="statusMessage">
           Total Logs gone through in last search: {*}<br /><input type="hidden" value="false" id="ifShowAll" /><input type="hidden" value="false" id="ifShowTestData" /></div>
 
                 
-            {*}
+            <br />
```

Comment:
I'll change this here, can you do a favor and check the other admin test files to see if there are more of this mistake? Thanks :+1: